pyaedt/generic/general_methods.py

```diff
@@ -1327,6 +1327,78 @@ def compute_fft(time_vals, value):  # pragma: no cover
     return freq, valueFFT
 
 
+@pyaedt_function_handler()
+def conversion_function(data, function_str=None):  # pragma: no cover
+    """This method performs various types of conversions on the input data based on the specified function string.
+    The available functions are:
+
+    - `"dB10"`: Converts the data to decibels using base 10 logarithm.
+    - `"dB20"`: Converts the data to decibels using base 20 logarithm.
+    - `"abs"`: Computes the absolute value of the data.
+    - `"real"`: Computes the real part of the data.
+    - `"imag"`: Computes the imaginary part of the data.
+    - `"norm"`: Normalizes the data to have values between 0 and 1.
+    - `"ang"`: Computes the phase angle of the data.
+    - `"ang_deg"`: Computes the phase angle of the data in degrees.
+
+    If an invalid function string is provided, the method returns ``False``.
```

Comment:
```suggestion
    If an invalid function string is specified, the method returns ``False``.
```
Changed to use the same term as in the method description (for reinforcement/consistency).